core/domain/voiceover_regeneration_services_test.py

```diff
@@ -590,3 +603,36 @@ def mock_synthesize_voiceover_for_html_string(
             errors_while_voiceover_regeneration,
             [('content_0', 'Mocked exception during voiceover regeneration')]
         )
+
+    def test_custom_rte_tags_sync_with_definitions(self) -> None:
+        with open(
+            feconf.RTE_EXTENSIONS_DEFINITIONS_PATH, 'r', encoding='utf-8'
+        ) as file:
+            file_contents = file.read()
+            match = re.search(
+                r'export default ({.*}) as const;', file_contents, re.DOTALL)
+
+            if match:
+                json_like_data = match.group(1)
+                custom_rte_tags_definition_dict = json.loads(json_like_data)
+
+                custom_rte_tag_to_names = {
```

Comment:
This shouldn't be defined in tests, you should derive this info from existing files. Otherwise it will fall out of sync.

See the notes in the comment above about using existing constants.